api/src/main/java/com/okta/authn/sdk/resource/FactorEnrollRequest.java

```diff
@@ -31,6 +31,5 @@
     FactorProvider getProvider();
     FactorEnrollRequest setProvider(FactorProvider provider);
 
-    FactorProfile getFactorProfile();
-    FactorEnrollRequest setFactorProfile(FactorProfile factorProfile);
+    FactorEnrollRequest setUserFactorProfile(UserFactorProfile userFactorProfile);
```

Comment:
my personal pref is to keep the old `setFactorProfile` name, 
If we do decouple the auth sdk from the mgmt one in the future the "UserProfile" term might have less meaning

non-blocking, take this with a grain of salt🧂

Reply:
ok, will stick to the name `setFactorProfile`.